sphere-common/src/main/java/io/sphere/sdk/models/LocalizedString.java

```diff
@@ -334,4 +334,8 @@ public String toString() {
             }
         };
     }
+
+    public static LocalizedString ofEnglish(final String translationForEnglish) {
```

Comment:
Personally I'm not very fan of aliases in class methods, because it makes me wonder what's the difference and I waste time checking the source, plus it makes the method lists larger. Unless there's a really good reason to do it, of course. I feel `ofEnglish` is good enough already, but it's up to you.